sentry-core/src/main/java/io/sentry/core/Breadcrumb.java

```diff
@@ -227,20 +227,7 @@ public void acceptUnknownProperties(@Nullable Map<String, Object> unknown) {
       clone.data = null;
     }
 
-    final Map<String, Object> unknownRef = unknown;
-    if (unknownRef != null) {
-      final Map<String, Object> unknownClone = new HashMap<>();
-
-      for (Map.Entry<String, Object> item : unknownRef.entrySet()) {
-        if (item != null) {
-          unknownClone.put(item.getKey(), item.getValue()); // shallow copy
-        }
-      }
-
-      clone.unknown = unknownClone;
-    } else {
-      clone.unknown = null;
-    }
+    clone.unknown = CollectionUtils.shallowCopy(unknown);
```

Comment:
more like a suggestion, what's about using this method and replace all the others shallow copy within clone methods? eg `Breadcrumb.data`

Reply:
Fixed in https://github.com/maciejwalkowiak/sentry-android/pull/1